libraries/AP_L1_Control/AP_L1_Control.cpp

```diff
@@ -127,6 +135,34 @@ float AP_L1_Control::turn_distance(float wp_radius, float turn_angle) const
     return distance_90 * turn_angle / 90.0f;
 }
 
+float AP_L1_Control::loiter_radius(const float radius) const
+{
+    // prevent an insane loiter bank limit
+    float sanitized_bank_limit = constrain_float(_loiter_bank_limit, 0.0f, 89.0f);
+    float lateral_accel_sea_level = tanf(radians(sanitized_bank_limit)) * GRAVITY_MSS;
```

Comment:
Whats the runtime cost of tanf? (I can't seem to actually find any documentation on it) This would get called at the navigation loop rate, so it would only be 10Hz, so I wasn't to concerned by the runtime cost.

I could instead push the sanitized_bank_limit and lateral_accel_sea_level calculations as statics, and then only recompute them if the parameter changed, but I was originally operating under the assumption that the (2x float + bool_ cost of RAM as well as the extra code was making it preferable to recalculate everytime. If this was incorrect I can convert it over to avoid the trig instead.